docs/content/en/docs/reference/changelog.md

```diff
@@ -10,7 +10,12 @@ menu:
 ## Unreleased
 
 ### Added
-- Add support for Kubernetes 1.22 [#1549](https://github.com/aws/eks-anywhere/pull/1549)
+
+## [v0.8.0](https://github.com/aws/eks-anywhere/releases/tag/v0.8.0)
+### Added
+- SSH keys and Users are now mutable [#1208](https://github.com/aws/eks-anywhere/issues/1208)
+- OIDC configuration is now mutable [#676](https://github.com/aws/eks-anywhere/issues/676)
+- Add support for Cilium's policy enforcement mode [#726](https://github.com/aws/eks-anywhere/issues/726)
 
```

Comment:
```suggestion
- Add support for Cilium's policy enforcement mode [#726](https://github.com/aws/eks-anywhere/issues/726)

### Changed
- Install Cilium networking through Helm instead of static manifest
```